docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
 
-Only superusers can set anonymization labels.
-To remove an anonymization label from an object, set it to `NULL`.
+NOTE: If no configuration is done, then by default no anonymization takes place. All users will have full access to table contents.
 
-Execute `SELECT * FROM diffix.show_labels();` to display the current labels in use by the extension.
+For more information about PostgreSQL security labels, see the official [documentation page](https://www.postgresql.org/docs/current/sql-security-label.html).
 
-Tables can be labeled as `public` or `personal`. Direct access is allowed to public data even for restricted users.
-Unlabeled tables can not be queried (unless `treat_unmarked_tables_as_public` is set to `true`).
+## Security labels
 
-Anonymization ID (AID) columns for a personal table have to be marked with the anonymization label `aid`.
-A personal table can have one or more AID columns.
+### Security labels for roles (users)
 
-In order to label a table as `personal` and AID columns, use the
-`diffix.mark_personal(namespace, table_name, aid_columns...)` procedure, for example:
+A user may have one of three security labels, `direct`, `anonymized_trusted`, and `anonymized_untrusted`. Users with security label `direct` have full access to the data: no anonymization takes place. The other two labels determine whether a user that is subject to anonymization is treated as _trusted_ or _untrusted_.
+
+* For trusted users, Diffix prevents _accidental_ release of personal data. So long as trusted users do not try to bypass Diffix anonymization, answers to queries are anonymous.
+* For untrusted users, Diffix prevents _intentional_ release of personal data. Even for users that are malicious and try to break Diffix anonymization, answer to queries are anonymous.
+
+Trusted users have fewer SQL restrictions than untrusted users, and therefore have better analytic utility.
+
+For example, the command to assign the security label `anonymized_untrusted` to the `public_access` is:
 
 ```SQL
-CALL diffix.mark_personal('public', 'my_table', 'id', 'last_name');
+SECURITY LABEL FOR pg_diffix ON ROLE public_access IS 'anonymized_untrusted';
 ```
 
-In order to label a table as `public`, use the `diffix.mark_public(namespace, table_name)` procedure.
+The value of the setting variable `pg_diffix.default_access_level` determines the default user class for unlabeled regular users (i.e. not superusers). It is set to `direct` by default.
+
+### Security labels for tables
+
+Tables may have one of two security labels, `public` or `personal`.
+
+* Tables labeled as `personal` are anonymized by the extension (for `anonymized_trusted` and `anonymized_untrusted` users, not `direct` users).
+* Tables labeled as `public` are not anonymized: all users have full access to these tables.
+
+The procedure `diffix.mark_public(namespace, table_name)` labels a table as `public`.
```

Comment:
Also needs AIDs `diffix.mark_public(namespace, table_name, id_column)`.

Reply:
Isn't it only `mark_personal` that requires AIDs?

Reply:
Oops, sorry, I thought this was mark personal!